dev/protos/update.sh

```diff
@@ -83,15 +83,15 @@ PBJS_ARGS=( --proto_path=. \
   --no-decode \
   --no-verify \
   --no-delimited \
-  --force-enum-string \
-  --force-number)
+  --force-enum-string)
```

Comment:
I was not able to get the ProtobufJS compiler to use "string|number" in its types. It only offers "Long", "Long|number" and "number". So I chose to use "Long|number" and replace it manually with "string|number". int32 (like nanoseonds) are not affected since they can be represented as number.